src/components/shared/IdleSearchField.js

```diff
@@ -91,13 +93,16 @@ export class IdleSearchField extends PureComponent<Props, State> {
     e.preventDefault();
   }
 
-  UNSAFE_componentWillReceiveProps(nextProps: Props) {
-    if (nextProps.defaultValue !== this.props.defaultValue) {
-      this._notifyIfChanged(nextProps.defaultValue || '');
-      this.setState({
-        value: nextProps.defaultValue || '',
-      });
+  static getDerivedStateFromProps(props: Props, state: State) {
+    if (
+      props.defaultValue !== state.previousDefaultValue &&
+      props.defaultValue !== state.value
+    ) {
+      return {
+        previousDefaultValue: props.defaultValue || '',
+      };
```

Comment:
The problem happens when the value is changed through the redux state. This happens especially when using the "back" button of the browser.
Previously you were changing the `value` in some cases, I think we should still do it, but you need to find the right condition.
Possibly you need 2 different conditions for `previousDefaultValue` and `value`.